08-walkthrough-3.Rmd

```diff
@@ -298,41 +320,52 @@ varImp(rf_fit_imp) %>%
 We can see whether these change with the different importance measures.
 
 ```{r}
-rf_fit_imp_permutation <- train(final_grade ~ .,
+rf_fit2_imp_permutation <- train(final_grade ~ .,
                 data = d_train,
                 method = "ranger",
                 importance = "permutation")
 
-varImp(rf_fit_imp_permutation) %>%
+varImp(rf_fit2_imp_permutation) %>%
     pluck(1) %>%
     rownames_to_column("var") %>%
     ggplot(aes(x = reorder(var, Overall), y = Overall)) +
     geom_col() +
     coord_flip()
 ```
 
-They are similar but somewhat different. One takeaway from this analysis is that what course students are in seems to have a different effect depending on the course. Also, how much students write in their discussion posts (`n`) seems to be very important - as does the time students spend in their course. Finally, there are some subject level differences (in terms of how predictive subject is). Perhaps grades should be normalized within subject: would this still be an important predictor, then?
+They are similar but somewhat different. One variable that shows up as important in both of the variable importance visualizations is `subjectFrScA`, a variable which means ___________________________.
+One takeaway from this analysis is that what course students are in seems to have a different effect depending on the course. Also, how much students write in their discussion posts (`n`) seems to be very important - as does the time students spend in their course. Finally, there are some subject level differences (in terms of how predictive subject is). Perhaps grades should be normalized within subject: would this still be an important predictor if we did that? We won't dive into that question here, but you can see how the line of research inquiry might progress as you start to explore your data with a machine learning model.
 
 ## Comparing a random forest to a regression
 
 You may be curious about comparing the predictive accuracy of the model to a linear model (a regression).
```

Comment:
say more! this is a really interesting sentence, and it would be great to spell out _why_ this is an interesting comparison, maybe even talking about when to choose ML over regression//vice versa

Reply:
Will think about this more and expand!